### ISSUE TITLE
Siri breaks over bluetooth via handfree car cmd

### ISSUE THREAD

**Issue by yaboijay2029** [checkra1n/BugTracker#491]
**Tell us about your setup:**
1. What iDevice are you using? 
iPhone 7+
2. On what version of iOS is it?
13.2.2
3. What version of checkra1n are you using? GUI or CLI?
0.9.5 GUI
4. What is your host system (OS version? Hackintosh? VM? etc.)?
Catalina
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
USB-C to A adapter
**What are the steps to reproduce the issue?**
1.  Jailbreak with Checkra1n
2. Use handsfree command on steering wheel
3. Siri splash screen with car logo shows but then immediately disappears
...

**What do you expect, and what is happening instead?**
For siri functionally via handsfree car command but it's not

**Does the issue also occur if you enable Safe Mode?**
Have no tried safemode but breaks in jailbreak mode with all tweaks disabled via iCleaner.

**Any other info, error logs, screenshots, ...?**

**Comment by vinnie12341234** [comment]
I have the exact same problem, whenever I hit the button programmed to bring up siri, siri comes on for a second or two and then goes away.  I've replicated this over bluetooth with both my truck and a Bose speaker.

iPhone X
iOS 13.2.2
0.9.5 GUI
Catalina
USB-A

Others on Reddit have the same issue, found here: https://www.reddit.com/r/jailbreak/comments/dwf7es/question_checkra1n_breaks_siri_through_bluetooth/?utm_source=share&utm_medium=ios_app&utm_name=iossmf

**Comment by vinnie12341234** [comment]
Is this a corner case or do you think it affects everyone?

**Comment by yaboijay2029** [comment]
It should affect everyone...i've tried everything but it only doesn't work until i'm in jailbreak mode. It works once after a respring and then after that it doesn't for some odd reason. Now there are times it does work but I've not done anything different to my phone.

**Comment by vinnie12341234** [comment]
So I can confirm that v0.9.6 does NOT fix this issue :(

**Comment by yaboijay2029** [comment]
Yeah I wish there was a way for it to be made as a known issue

**Comment by Grisu70** [comment]
Same problem here

IOS Version 13.2.2 / 13.2.3
Checkra1n version 0.9.6 beta
Iphone X 64GB

I'm facing the problem with my Cellularline Interphone for helmet.

Problem disappear if i reboot with no checkra1n

B. regards
Fabrizio

**Comment by vinnie12341234** [comment]
I don't want to jinx it, but I upgraded to iOS 13.3 and for 2 straight commutes, no issues with Siri over bluetooth.  @yaboijay2029  @Grisu70 you should upgrade and try it.   You'll need to make a change in Checkra1n 0.9.6 to make it work with 13.3.   If you need help, ping me on reddit (user vinnie12341234).

**Comment by Grisu70** [comment]
How to ping you in reddit?
Is 13.3 ok for jailbreak? I read Checkra1n has issue with icloud...

Inviato da iPhone

**Comment by vinnie12341234** [comment]
@Grisu70 don't bother, 0.9.7 added official support for 13.3.   I'm about to try it and will post later today my results .

**Comment by vinnie12341234** [comment]
Ever since iOS 13.3, the issue has gone away for me!

**Comment by yaboijay2029** [comment]
Same here bro. Looks like all is well!

**Comment by Grisu70** [comment]
Hi,
not had the time to test completely with my helmet (i had biggest 
problem with it)
in the car, things seems get better , in the sense that now siri always 
reply, but the issue is that most of the times while i'm dictating a 
message, it stops listening, and the message text is not complete.

Fabrizio

Il 16/12/2019 19:15, yaboijay2029 ha scritto:

**Comment by vinnie12341234** [comment]
So, I'm back to report that even with iOS 13.3 and 0.9.7 the issue persists.  It did go away for a few days but now is back.

**Comment by vinnie12341234** [comment]
@Siguza any chance you can look at this please?

**Comment by computersteve** [comment]
I am having the same problem With iOS 13.3.1 & latest 0.9.8.. Tried everything like SiriUnlock & Pass-by but every time I press my cars Siri button it pop up for a few seconds then cancels out..

**Comment by Grisu70** [comment]
Still have same problem here

Inviato da iPhone

**Comment by computersteve** [comment]
Yes, Same Problem & doing a full restore / setup as new on both iOS 13.3 / 13.3.1 didn't do anything to resolve it.

**Comment by yaboijay2029** [comment]
I had to take a break from checkra1n because of this issue. It never was an issue on Unc0ver so i wonder what the difference is

**Comment by vinnie12341234** [comment]
@yaboijay2029 Right but I don't think UnC0ver has been updated for A11 devices yet, I know it will be soon.  Are you saying you switched to UnC0ver and you no longer have the issue?

**Comment by yaboijay2029** [comment]
I have switched to Unc0ver as of today because it was updated for A8-A11 devices. I will give it a shot today.

**Comment by Grisu70** [comment]
Hi,
anyway i fund that Siri is quite buggy in IOS 13.
even without jailbreak, some time it does not let me finish my prhase, or does not recognise some command (like append, or read again).
And this is happening only on all bluetoot headphone, or helmet except AirPods.
the same headphone or bt helmet, worked like a charm in iOS 12

I think that it is a mixture of IOS 13 bug and checkra1in

Fabrizio

**Comment by yaboijay2029** [comment]
has anyone tested this issue on the latest version of checkra1n?

**Comment by vinnie12341234** [comment]
@yaboijay2029 Thanks for getting back to us.  Yes, I have tested on the latest checkra1n 0.10.1 beta

**Comment by yaboijay2029** [comment]
Yeah i've been busy with work. I wish there was a way to debug this ya know.  

I'm assuming it's still messing up for you? @vinnie12341234 

I've been using unc0ver 4.3 since it came out. It's a hit or miss honestly still.
